pkg/operator/etcd/pod.go

```diff
@@ -108,6 +112,14 @@ func UpdatePod(obj *corev1.Pod, spec *Spec) {
 	// Update labels, but ignore existing ones we don't set.
 	update.Labels(&obj.Labels, spec.Labels)
 
+	// Update annotations and recreate pod if annotations are updated or removed.
+	annotations := spec.Annotations
+	annotationsHash := map[string]string{
+		"annotationsDeltaHash": annotationHash(annotations),
```

Comment:
Annotation keys that our operator sets should be prefixed with `planetscale.com/` to avoid overlap with annotations that other entities set. Also the convention is to use dashes to separate lowercase words, so we can call this `planetscale.com/annotation-keys-hash`.